apps/aehttp/priv/swagger.json

```diff
@@ -128,6 +128,12 @@
               "$ref" : "#/definitions/Block"
             }
           },
+          "400" : {
```

Comment:
Responses appears missing for POST /tx in swagger.json: issue in generation from YAML? inconsistent YAML indentation? something I am missing?

Reply:
Yes, there had been an identation issue in the yaml file. Patched now